devices/common_patches/h7_dbgmcu_dualcore.yaml

```diff
@@ -0,0 +1,170 @@
+          bitOffset: 23
+          bitWidth: 1
+        I2C2:
+          description: I2C2 SMBUS timeout stop when Cortex-M4 in debug mode
+          bitOffset: 22
+          bitWidth: 1
+        I2C1:
+          description: I2C1 SMBUS timeout stop when Cortex-M4 in debug mode
+          bitOffset: 21
+          bitWidth: 1
+        DBG_WWDG2:
+          description: WWDG2 stop in when Cortex-M4 when Cortex-M4 in debug mode
+          bitOffset: 11
+          bitWidth: 1
+        LPTIM1:
+          description: LPTIM1 stop when Cortex-M4 in debug mode
+          bitOffset: 9
+          bitWidth: 1
+        TIM14:
+          description: TIM14 stop when Cortex-M4 in debug mode
+          bitOffset: 8
+          bitWidth: 1
+        TIM13:
+          description: TIM13 stop when Cortex-M4 in debug mode
+          bitOffset: 7
+          bitWidth: 1
+        TIM12:
+          description: TIM12 stop when Cortex-M4 in debug mode
+          bitOffset: 6
+          bitWidth: 1
+        TIM7:
+          description: TIM7 stop when Cortex-M4 in debug mode
+          bitOffset: 5
+          bitWidth: 1
+        TIM6:
+          description: TIM6 stop when Cortex-M4 in debug mode
+          bitOffset: 4
+          bitWidth: 1
+        TIM5:
+          description: TIM5 stop when Cortex-M4 in debug mode
+          bitOffset: 3
+          bitWidth: 1
+        TIM4:
+          description: TIM4 stop when Cortex-M4 in debug mode
+          bitOffset: 2
+          bitWidth: 1
+        TIM3:
+          description: TIM3 stop when Cortex-M4 in debug mode
+          bitOffset: 1
+          bitWidth: 1
+        TIM2:
+          description: TIM2 stop when Cortex-M4 in debug mode
+          bitOffset: 0
+          bitWidth: 1
+    APB2FZ2:
+      description: APB2 peripheral freeze register CPU2
+      addressOffset: 0x048
+      access: read-write
+      resetValue: 0x00000000
+      fields:
+        HRTIM:
+          description: HRTIM stop when Cortex-M4 in debug mode
+          bitOffset: 29
+          bitWidth: 1
+        TIM17:
+          description: TIM17 stop when Cortex-M4 in debug mode
+          bitOffset: 18
+          bitWidth: 1
+        TIM16:
+          description: TIM16 stop when Cortex-M4 in debug mode
+          bitOffset: 17
+          bitWidth: 1
+        TIM15:
+          description: TIM15 stop when Cortex-M4 in debug mode
+          bitOffset: 16
+          bitWidth: 1
+        TIM8:
+          description: TIM8 stop when Cortex-M4 in debug mode
+          bitOffset: 1
+          bitWidth: 1
+        TIM1:
+          description: TIM1 stop when Cortex-M4 in debug mode
+          bitOffset: 0
+          bitWidth: 1
+    APB4FZ2:
+      description: APB4 peripheral freeze register CPU2
+      addressOffset: 0x058
+      access: read-write
+      resetValue: 0x00000000
+      fields:
+        LSDWDG2:
+          description: LS watchdog for D2 stop when Cortex-M4 in debug mode
+          bitOffset: 19
+          bitWidth: 1
+        LSDWDG1:
```

Comment:
The RM calls this `WDGLSD1` instead of `LSDWDG1`

Reply:
Resolved to the RM `WDGLSD1`